search/django/weedid/views.py

```diff
@@ -62,6 +62,16 @@ def upload(request):
         upload_dir, upload_id = setup_upload_dir(os.path.join(UPLOAD_DIR, str(user.id)))
         weedcoco_path = store_tmp_weedcoco(file_weedcoco, upload_dir)
         create_upload_entity(weedcoco_path, upload_id, user.id)
+    except JsonValidationError as e:
+        traceback.print_exc()
+        return HttpResponseForbidden(
```

Comment:
I'd prefer HTTP 400 than 403. 403 is quite specifically about preventing access to a resource. We will want this response to be a JSON blob.